app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value():
-    pass
+    parts = split_integer(17, 4)
+    assert sum(parts) == 17
 
```

Comment:
Code Style
Don't create unnecessary variables for simple tests:
Good example:

def test_():
    assert some_function_to_test() == [10]
    
 Don't forget to check [checklist ](https://github.com/mate-academy/py-split-integer/blob/master/checklist.md)before you pushing your solution
 assert sum(split_integer(17, 4) == 17